x-pack/plugins/serverless_search/public/layout/nav.tsx

```diff
@@ -17,12 +17,12 @@ import type { ServerlessPluginStart } from '@kbn/serverless/public';
 import type { CloudStart } from '@kbn/cloud-plugin/public';
 
 // Hiding this until page is in a better space
-const _connectorItem = {
-  link: 'serverlessConnectors',
-  title: i18n.translate('xpack.serverlessSearch.nav.connectors', {
-    defaultMessage: 'Connectors',
-  }),
-};
+// const _connectorItem = {
+//   link: 'serverlessConnectors',
+//   title: i18n.translate('xpack.serverlessSearch.nav.connectors', {
+//     defaultMessage: 'Connectors',
+//   }),
+// };
```

Comment:
What's the reason for this change?

Reply:
I think it was a bit of unused code, and typescript fails on unused variables. I'll remove the whole commented section.

Reply:
oh, I see what you mean - I think the code that split the changes from the branch sometimes reverted some of the changes that since been done in main.

Reply:
There's a comment above this that says "hiding this until page is in a better space". What's the reason for removing this instead of just commenting it out and leaving it in place, particularly with pushing through this PR without waiting for an approval from the code owners?

Reply:
@sphilipse We should have waited for owner approval, so sorry about that. Leaving unused code around, commented or not, is unnecessary though and it will be available in the git history if required in the future.

Reply:
Of course it's available, which is why this is not that big a big deal. But there are also reasons why we keep things around, commented or not, and we generally don't want to break teams' internal processes without good reason.

Reply:
@sphilipse As far as I can see this was "hidden" in 5ebba533ba36eff372bef05ee0fe76377a0b2542. Normally you should have either removed the code, or if it was important to keep in-line, just commented it out in-place with a message above it on why it was commented out. The dummy `const` that was added produced TypeScript errors which is why we had to fix it. I hope you appreciate why we had to move quickly on this one, so please bear that in mind in the discussion